lib/node_modules/@stdlib/stats/base/dists/rayleigh/logcdf/src/main.c

```diff
@@ -0,0 +1,60 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/stats/base/dists/rayleigh/logcdf.h"
+#include "stdlib/math/base/special/expm1.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/log1p.h"
+#include "stdlib/math/base/special/exp.h"
+#include "stdlib/math/base/special/pow.h"
+#include "stdlib/math/base/special/ln.h"
+#include "stdlib/constants/float64/ln_half.h"
+#include "stdlib/constants/float64/ninf.h"
+
+
+/**
+* Evaluates the logarithm of the cumulative distribution function (CDF) for a Rayleigh distribution with scale parameter `sigma` at a value `x`.
+*
+* @param x        input value
+* @param sigma    scale parameter
+* @return         evaluated LOGCDF
```

Comment:
To match `native.js` return value description: 

```suggestion
* @return         evaluated logCDF
```